include/common/mir/logging/logger.h

```diff
@@ -62,6 +63,7 @@ class Logger
 
 void log(Severity severity, const std::string& message, const std::string& component);
 void set_logger(std::shared_ptr<Logger> const& new_logger);
+void format_message(std::ostream&, Severity, const std::string&, const std::string&);
```

Comment:
The parameters in an interface should probably be named :wink: 

Also, we spell `const std::string&` as `std::string const&`